<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>UserHistoryCfgBase</class>
  <widget class="QWidget" name="UserHistoryCfg" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>285</width>
        <height>145</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item row="0" column="2" >
        <widget class="QSpinBox" name="edtSize" >
          <property name="maximum" stdset="0" >
            <number>1000</number>
          </property>
        </widget>
      </item>
      <item row="0" column="3" >
        <widget class="QLabel" name="lblSize1" >
          <property name="text" stdset="0" >
            <string>Mb</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="QCheckBox" name="chkSize" >
          <property name="text" stdset="0" >
            <string/>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QLabel" name="lblSize" >
          <property name="text" stdset="0" >
            <string>Max history file size</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="2" >
        <widget class="QSpinBox" name="edtDays" >
          <property name="maximum" stdset="0" >
            <number>1000</number>
          </property>
        </widget>
      </item>
      <item row="1" column="3" >
        <widget class="QLabel" name="lblDays1" >
          <property name="text" stdset="0" >
            <string>days</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="QLabel" name="lblDays" >
          <property name="text" stdset="0" >
            <string>Keep history </string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QCheckBox" name="chkDays" >
          <property name="text" stdset="0" >
            <string/>
          </property>
        </widget>
      </item>
      <item row="0" column="4" >
        <spacer name="Spacer2" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="2" column="1" >
        <spacer name="Spacer5" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QWidget</class>
      <extends>QWidget</extends>
      <header location="global" >QWidget</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
